docs/add-ins/addin-tutorial.md

```diff
@@ -141,80 +253,63 @@ Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this li
+    ```
+
 1. Add the following elements as children of the `bt:LongStrings` element.
 
-   ```xml
-    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your Gists and allows you to insert their contents into the current message">
-      <bt:Override Locale="es-ES" Value="Muestra una lista de sus Gists y permite insertar su contenido en el mensaje actual"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the Gist you mark as default into the current message">
-      <bt:Override Locale="es-ES" Value="Inserta el contenido de lo Gist que marca como predeterminado en el mensaje actual"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your gists and allows you to insert their contents into the current message"/>
+    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the gist you mark as default into the current message"/>
+    ```
+
+1. Save your changes to the manifest. 
 
-This defines the string values that will be used for the add-in. It also localizes the strings into Spanish by providing a `bt:Override` element for each string. Additional languages can be added as additional `bt:Override` elements. The `DefaultValue` is used for clients that use the locale specified in the `DefaultLocale` element in the manifest.
 
-Save your changes to the manifest. Since we installed the add-in from a file, we need to reinstall it in order for the changes to take effect.
+### Reinstall the add-in
 
-1. Open Outlook 2016. On the **Home** tab in the ribbon, click the **Store** button.
+Since you previously installed the add-in from a file, you must reinstall it in order for the manifest changes to take effect. 
 
-1. Click the **My add-ins** link on the left side.
+1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to locate the **Custom add-ins** section at the bottom of the **My add-ins** dialog box. 
 
-1. Click the **...** button next to the **Git the Gist** entry, then choose **Remove**.
+1. Choose the **...** button next to the **Git the gist** entry and then choose **Remove**.
 
-1. Close the Store window.
+1. Close the **My add-ins** window.
 
 1. The custom button should disappear from the ribbon momentarily.
 
-1. Reinstall the add-in using the new manifest.
+1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to reinstall the add-in using the updated **manifest.xml** file.
 
-Now when you compose a new message in Outlook, you should see two buttons on the ribbon: **Insert Gist** and **Insert Default Gist**. Now we can work on implementing the add-in functionality.
+After you've completed these steps, you should the two new buttons when you compose a new message in Outlook: **Insert gist** and **Insert default gist**. 
```

Comment:
I'm unclear about what this should say. Maybe "...you should then see two new..."

Reply:
Good catch; I've updated this sentence for clarity.